makefiles/tools/openocd.inc.mk

```diff
@@ -20,6 +20,9 @@ OPENOCD_CONFIG ?= $(BOARDDIR)/dist/openocd.cfg
 OPENOCD_TARGETS = debug% flash% reset
 $(call target-export-variables,$(OPENOCD_TARGETS),OPENOCD_CONFIG)
 
+# Export OPENOCD_CORE to required targets
+$(call target-export-variables,$(OPENOCD_CORE),OPENOCD_CORE)
```

Comment:
I would keep them in OPENOCD_TARGETS because you could have multi-core architecture with flash (stm32h7 ?) and then use OPENOCD_CORE with flash and debug targets.